custom-connectors/ZellisIntelligencePlatform/README.md

```diff
@@ -0,0 +1,44 @@
+## Zellis Intelligence Platform Connector
+An official connector working with the Zellis Intelligence Platform.
```

Comment:
@basha-k If you want to add your connector in custom-connector folder and do not want to certify, please apply all the changes suggested above.